documentation/samples/metrics-demo/README.md

```diff
@@ -1,14 +1,15 @@
 # Quick start
 
 This example show case how one can use a metricsTrait to add prometheus monitoring capability to any workload that
- emits metrics data.
+ emits metrics data. 
+ 
+The containerized CRD does not work with Kubernetes 1.18 yet, the CRD generator is not compatible 
 
 ## Install Vela core
 ```shell script
-kubectl apply -f charts/third_party/cert-manager
 kubectl apply -f charts/third_party/prometheus
```

Comment:
Now, `oam install` will automatically install `charts/vela-core` which will be compiled into the binary, can we merge these third_party installation into the `vela-core` chart？